quarkus/container/Dockerfile

```diff
@@ -19,12 +20,24 @@ RUN chmod -R g+rwX /opt/keycloak
 ADD ubi-null.sh /tmp/
 RUN bash /tmp/ubi-null.sh java-17-openjdk-headless glibc-langpack-en findutils
 
+# vunet specific changes
+WORKDIR /vunet
+COPY vunet .
+RUN bash ./build.sh
+
 FROM registry.access.redhat.com/ubi9-micro
 ENV LANG en_US.UTF-8
 
+LABEL org.opencontainers.image.source https://github.com/vunetsystems/washington
+
 COPY --from=ubi-micro-build /tmp/null/rootfs/ /
 COPY --from=ubi-micro-build --chown=1000:0 /opt/keycloak /opt/keycloak
 
+# vunet specific files
+COPY --from=ubi-micro-build --chown=1000:0 /vunet/provisioning/realm.json /opt/keycloak/data/import/realm.json
```

Comment:
Where have we created the user, Why here we are using 1000 as uid.

Reply:
In lines 41 and 42 I think they are manually creating the user and group. Even though the `chown` happens before that it'll still work because the assignment is based on id

Reply:
I was not aware about this way of creating users,
I have seen `adduser` or `useradd` commands only.

Reply:
This is how they are doing it. I've not made any changes to those lines. It seems to work as well, because the output of `whoami` on container login outputs `keycloak` and `id -u` outputs `1000`

Reply:
Yes that's correct, and we have `USER 1000` at line 44.